docs/en/operations/backup.md

```diff
@@ -356,6 +357,267 @@ But keep in mind that:
+                <secret_access_key>Your AWS Secret Key</secret_access_key>
+                <!-- highlight-end -->
+                <s3_max_single_part_upload_size>33554432</s3_max_single_part_upload_size>
+            </s3_backup_compact>
+            <s3_backup_wide>
+                <type>s3_plain</type>
+                <!-- highlight-start -->
+                <!-- NOTE: /backup_wide/ is a name of BACKUP -->
+                <endpoint>https://s3-backup-for-read-only-table.s3.amazonaws.com/tables/backup_wide/</endpoint>
+                <access_key_id>Your AWS Access Key</access_key_id>
+                <secret_access_key>Your AWS Secret Key</secret_access_key>
+                <!-- highlight-end -->
+                <s3_max_single_part_upload_size>33554432</s3_max_single_part_upload_size>
+            </s3_backup_wide>
+        </disks>
+        <policies>
+            <s3_backup_compact>
+                <volumes>
+                    <main>
+                        <disk>s3_backup_compact</disk>
+                    </main>
+                </volumes>
+            </s3_backup_compact>
+            <s3_backup_wide>
+                <volumes>
+                    <main>
+                        <disk>s3_backup_wide</disk>
+                    </main>
+                </volumes>
+            </s3_backup_wide>
+        </policies>
+    </storage_configuration>
+    <backups>
+        <allowed_disk>backup_disk_s3_plain</allowed_disk>
+    </backups>
+</clickhouse>
+```
+
+### Create a database of type ordinary
+
+Enable the use of ordinary databases:
+```sql
+set allow_deprecated_database_ordinary=1;
+```
+
+Create the database:
+```sql
+create database ordinary_db engine=Ordinary;
+```
+
+### Create a table
+
+This will create a table in the `ordinary_db` database that has five partitions:
+- there are 100 rows, with two columns `part` and `key`
+- `key` is set to the value provided by the function `numbers(100)`
+- `part` is set to the value `number % 5`
+- the table is partitioned on the column `part`
+
+```sql
+CREATE TABLE ordinary_db.compact
+ENGINE = MergeTree
+PARTITION BY part
+ORDER BY key AS
+SELECT
+    number % 5 AS part,
+    number AS key
+FROM numbers(100)
+```
+
+### Backup the table to S3
+```sql
+BACKUP TABLE ordinary_db.compact
+TO Disk('backup_disk_s3_plain', 'backup_compact')
+SETTINGS deduplicate_files = 0
+```
+```response
+┌─id───────────────────────────────────┬─status─────────┐
+│ 6b323487-87be-40f6-99ac-3d049221b98a │ BACKUP_CREATED │
+└──────────────────────────────────────┴────────────────┘
+
+1 row in set. Elapsed: 5.698 sec.
+```
+
+### Examine the way data is stored in S3
+
+Navigate to the bucket and locate the folder for the backup.
+
+![folder contents in S3](@site/docs/en/operations/images/backup-for-RO-attach-one-partition.png)
+
+### Drop the table
+```sql
+DROP TABLE ordinary_db.compact
+```
+
+### Attach the backup
+```sql
+ATTACH TABLE ordinary_db.compact
```

Comment:
Actually type of the parts is not important after everything had been fixed, so you can remove mentioning of the `compact`/`wide` here